### ISSUE TITLE
Helm Chart - Extra guardrails to create the ServiceMonitor

### ISSUE THREAD

**Issue by cmontemuino** [kube-logging/logging-operator#1630]
**Is your feature request related to a problem? Please describe.**

Enabling the servicemonitor is assuming Prometheus is installed already.

**Describe the solution you'd like**

Change https://github.com/kube-logging/logging-operator/blob/2c4ff95bd1a0dc05bcb092ca9018da5d38ff897d/charts/logging-operator/templates/service_monitor.yaml#L1 to:

```yaml
{{ if and (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") .Values.monitoring.serviceMonitor.enabled }}
```

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by cmontemuino** [comment]
Is it ok if I submit a PR for this?

**Comment by pepov** [comment]
@cmontemuino of course, just please make sure you add this for every occurence where we install a servicemonitor or any other prometheus resource

**Comment by cmontemuino** [comment]
@pepov: Confirmed the change is required in one single place only.

https://github.com/kube-logging/logging-operator/pull/1633
